dev/com.ibm.ws.cdi.visibility_fat/.classpath

```diff
@@ -8,7 +8,7 @@
 	<classpathentry kind="src" path="test-applications/visTestEjbAsEjbLib.jar/src"/>
 	<classpathentry kind="src" path="test-applications/visTestEjbWarLib.jar/src"/>
 	<classpathentry kind="src" path="test-applications/visTestWarLib.jar/src"/>
-	<classpathentry kind="src" path="test-applications/maskedClass.ear/src"/>
+	<classpathentry kind="src" path="test-applications/maskedClass.ear/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.

Reply:
I have seen it both ways.  I am glad to not have resources as well.  I will change that and the other ones that do that.

Reply:
I have put in the changes, but left the ones in microprofile.config_fat since they do resources in that .classpath so it matches what is done for other test-applications.